dotnet-local.sh

```diff
@@ -2,9 +2,11 @@
 ROOT="$(dirname "${BASH_SOURCE}")"
 FULLROOT="$(cd "${ROOT}"; pwd)"
 if [[ -x "${ROOT}/bin/Release/dotnet/dotnet" ]]; then
-    DOTNETSDK_WORKLOAD_MANIFEST_ROOTS=${FULLROOT}/bin/Release/lib/sdk-manifests DOTNETSDK_WORKLOAD_PACK_ROOTS=${FULLROOT}/bin/Release/lib exec ${ROOT}/bin/Release/dotnet/dotnet "$@"
+    DOTNET_ROOT=${FULLROOT}/bin/Release/dotnet
```

Comment:
This should probably use `export`: https://superuser.com/questions/18988/difference-between-a-b-and-export-a-b-in-bash

`export` wasn't previously needed because all the environment variables were part of the command execution.

For example, this:

```sh
A=1 B=2 C=3 child-process
```

is equivalent to:

```sh
(
  export A=1
  export B=2
  export C=3
  child-process
)
```

Additional, for greater sanity we should just make everything a separate `export`, and leave the `exec …/dotnet` invocation on its own line.